shell/packages/sandstorm-ui-topbar/topbar.html

```diff
@@ -40,9 +40,15 @@
         {{/unless}}
       {{/each}}
     </ul>
-    <ul class="navbar {{#unless showNavbar}}hide-desktop{{/unless}}">
+    <ul class="navbar {{#if hideNavbar}}hide-desktop{{else}}{{#if shrinkNavbar}}shrink-desktop{{/if}}{{/if}}">
       <li class="navitem-create-grain {{#if isCurrentRoute 'apps'}}current{{/if}}"><a href="{{pathFor route='apps'}}">Apps</a></li>
       <li class="navitem-open-grain {{#if isCurrentRoute 'grains'}}current{{/if}}"><a href="{{pathFor route='grains'}}"><span>Grains</span></a></li>
+
+      <!-- TODO(cleanup): ul's can only contain li's, so stuff below here is not really valid -->
+      <button class="navbar-shrink">
```

Comment:
... Apparently not. Not sure why I didn't. Fixed.

(I think I was distracted by the non-wrapped `<ul>` below it, or something.)